GoogleSignIn/Sources/GIDEMMErrorHandler.m

```diff
@@ -90,7 +90,12 @@ - (BOOL)handleErrorFromResponse:(NSDictionary<NSString *, id> *)response
   }
   // All UI must happen in the main thread.
   dispatch_async(dispatch_get_main_queue(), ^() {
-    UIWindow *keyWindow = [UIApplication sharedApplication].keyWindow;
+    UIWindow *keyWindow = [self keyWindow];
+    if (!keyWindow) {
+      // Shouldn't happen, just in case.
+      completion();
+      return;
+    }
     CGRect keyWindowBounds = CGRectIsEmpty(keyWindow.bounds) ?
```

Comment:
Hi @XiangtianDai , Could we also support scene-based lifecycle here ?  Which need connect UIWindow to a specific UIWindowScene.

Here is an example of my PR #92

Reply:
@lzw-all-in Updated the code to use `initWithWindowScene:` if available -- does this look good to you?

Reply:
@XiangtianDai are we ready to resolve this thread?

Reply:
I believe my updated code addresses comments from @lzw-all-in